pylzy/lzy/whiteboards/index.py

```diff
@@ -67,7 +68,10 @@ async def __start(self) -> None:
         global CHANNEL
         if not CHANNEL:
             CHANNEL = build_channel(
-                endpoint, interceptors=add_headers_interceptor({"authorization": f"Bearer {token}"})
+                endpoint, interceptors=add_headers_interceptor({
```

Comment:
I think we should move call `add_headers_interceptor({..})` into some `lzy.utils.grpc` module as some `build_headers(token)` function;
This way you will get rid of repentance and incapsulate version logic and string constants.